spec/requests/posts_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
-RSpec.describe "Posts", type: :request do
-  describe "GET /index" do
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
     pending "add some examples (or delete) #{__FILE__}"
   end
 end
```

Comment:
- [ ] I cannot find any relevant spec tests for the post index and post show requests. This file should contain your posts tests, rather it contains dummy pending test generated with Rails. Kindly add specs here for index and show requests.

- [ ] Kindly ensure you cover the required scenarios, like rendering template, checking for placeholder, and maybe HTTP status.

Reply:
I made a mistake and submitted the wrong part of the project, this PR is for the controllers themselves not the specs. Apologies